docs/section9/Rule6-w.md

```diff
@@ -1,44 +1,37 @@
+**Appendix G Section Reference:**  
 
-**Applicability:** All required data elements exist for P_RMR and B_RMR  
+**Applicability:** All required data elements exist for P_RMR  
 **Applicability Checks:** None  
 **Manual Check:** Yes  
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
 ## Rule Logic: 
 
-- Check if each space has window or skylight in the building segment in the Proposed model: ```For building_segment in P_RMR.building.building_segments:```  
+- Check if each space has window or skylight in the building segment in the Proposed model: ```For building_segment_p in P_RMR.building.building_segments:```  
 
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
+  - For each thermal_block in building segment: ```thermal_block_p in building_segment_p.thermal_blocks:```  
 
-  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
+  - For each zone in thermal block: ```zone_p in thermal_block_p.zones:```  
 
-  - Get space from thermal zone: ```space in thermal_zone.spaces:```  
+    - Get surfaces in zone: ```surface_p in zone_p.surfaces```  
 
-  - Get surfaces in space: ```surface in space.surfaces```  
+      - Check if the surface is exterior and has openings: ```if ( surface_p.adjacent_to == AMBIENT ) AND ( COUNT( surface_p.fenestration_subsurfaces ) >= 1 ): daylight_flag_p == TRUE```  
 
-    - Check if the surface is exterior and has openings: ```if ( surface.adjacent_to in [AMBIENT] ) AND ( surface.fenestration_subsurfaces ): daylight_flag_proposed == 1```  
+    -  For each space in zone: ```space_p in zone_p.spaces:```  
+      - Get interior_lighting in space: ```interior_lighting_p = space_p.interior_lighting```  
 
-    **Rule Assertion:** For each space in the Proposed model:  
+        - Check if any interior_lighting has daylight control: ```if ( lighting.has_daylight_control  == TRUE for lighting in interior_lighting_p ): has_daylight_control_flag == TRUE```  
 
-    - Case 1, the space has window or skylight and daylight control is modeled: ```daylight_control_flag == 1 AND space.has_daylight_control == 1: PASS```  
+        **Rule Assertion:** For each zone in the Proposed model:  
 
-    - Case 2, the space has window or skylight and daylight control is not modeled:  ```daylight_control_flag == 1 AND space.has_daylight_control == 0: CAUTION```
+        - Case 1, the zone has window or skylight and daylight control is modeled: ```daylight_control_flag == TRUE AND has_daylight_control_flag == TRUE: PASS```  
 
-    - Case 3, the space does not have window or skylight and daylight control is modeled: ```daylight_control_flag == 0 AND space.has_daylight_control == 1: FAIL```
+        - Case 2, the zone has window or skylight and daylight control is not modeled:  ```daylight_control_flag == TRUE AND has_daylight_control_flag == FALSE: CAUTION```  
 
```

Comment:
Question from @xingcx 
1. Do we need to check schedule adjustment in the RDS?
2. As per Section 9.4.1(e) and (f), some spaces with window/skylight are not required to have daylight sensors. Do we want to exempt these spaces in the RDS?  e.g. wattage in primary daylight/secondary daylight/top daylight area.

Reply
1. TBD
2. No, this was discussed by the schema WG and we decided against that. If a space has windows but no daylighting control, then we simply flag it and the additional tests have to be determined by manual review.